.github/workflows/main.yml

```diff
@@ -0,0 +1,66 @@
+---
+name: hs-dbus-signature CI
+
+# yamllint disable-line rule:truthy
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # MANDATORY CHECKS THAT USE TOX
+  tox-checks:
+    strategy:
+      matrix:
+        include:
+          - task: make -f Makefile lint
+          - task: make -f Makefile test
+          - task: make -f Makefile coverage
```

Comment:
I think this would make more sense as:
```
matrix:
  task: [lint, test, coverage]
```
`include` is only useful when you want to specify a set of specific instructions as opposed to simply iterating through all of the provided lists. Given that there's only one list, we should do the above.